packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -105,7 +65,7 @@ export const ConnectedServices = () => {
       data-testid="settings-connected-services"
     >
       <h2 className="font-header font-bold ml-4 mb-4">Connected Services</h2>
-      <div className="bg-white tablet:rounded-xl shadow px-6 pt-7 pb-9">
+      <div className="bg-white tablet:rounded-xl shadow px-6 pt-7 pb-8">
```

Comment:
Looks like the padding here does not jive with the rest of the mobile padding

<img width="410" alt="Screen Shot 2020-10-07 at 3 34 57 PM" src="https://user-images.githubusercontent.com/6392049/95379069-da9f3580-08b2-11eb-825b-5dcf3759e47e.png">

Reply:
nice catch, fixed this up